### DISCUSSION TITLE
How to customize Target banners?

### DISCUSSION THREAD

**Discussion by sharpninja** [nuke-build/nuke#837] [Q&A] [answered]
Is there a way to customize target banners (or suppress them) without rebuilding all of nuke?

**Comment by matkoch** [comment]
I already answered on Slack :) No, and also not planned.

**Reply by sharpninja** [reply]
![image](https://user-images.githubusercontent.com/16146732/148597221-167dea34-7272-4b1a-8b49-52fdfc14f1f1.png)

I'm not a regular user of Slack and the above is what I see.  I did not know I had to click anything to see the reply.

**Comment by sharpninja** [comment, marked as answer]
I have added two attributes to suppress banners on targets and the rest of the build.  The PR is: https://github.com/nuke-build/nuke/pull/887#issue-1167181944

I am completely open to suggestions for refining the submission.
